library/src/main/java/org/jellyfin/apiclient/interaction/websocket/ApiWebSocket.java

```diff
@@ -290,63 +287,30 @@ private void OnGeneralCommand(String json)
                 String text = args.getCommand().getArguments().get("Text");
                 String timeoutMs = args.getCommand().getArguments().get("TimeoutMs");
 
-                long expected = 0;
-                tangible.RefObject<Long> tempRef_expected = new tangible.RefObject<Long>(expected);
-                LongHelper.TryParseCultureInvariant(timeoutMs, tempRef_expected);
-                expected = tempRef_expected.argValue;
-
                 MessageCommand command = new MessageCommand();
-
                 command.setHeader(header);
                 command.setText(text);
-                command.setTimeoutMs(expected);
+                command.setTimeoutMs(Long.parseLong(timeoutMs));
```

Comment:
We should probably catch `NumberFormatException`s here like the `TryParseCultureInvariant` method did.

Reply:
I think the best short term plan is to pass the exception up a level so the app can handle the error with its chosen logging implementation.